### DISCUSSION TITLE
YMCD人格分类 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#477] [General]
# YMCD人格分类 - Miao Yu | 于淼

https://yufree.cn/cn/2024/09/21/ymcd/

**Comment by CyrusYip** [comment]
> D代表Design，其反面是Free。

这两个单词词性不一致，后者可以改成 Freedom，这样就都是名词了。

---

> 我现在就给设计一套类似系统，暂且命名为YMCD（Young Man Che Dan）系统

小伙子扯淡，哈哈。

**Reply by yufree** [reply]
听劝，改。

**Reply by CyrusYip** [reply]
突然想到 YMCD 的全称可以是 Yu Miao Che Dan。
